ch04packaging/03Packaging.ipynb.py

```diff
@@ -468,16 +507,16 @@ def process():
 Portions of the material are taken from [Software Carpentry](http://software-carpentry.org/)
 
 # %% [markdown]
-# You may well want to formalise this using the [codemeta.json](https://codemeta.github.io/) standard or the [citation file format](http://citation-file-format.github.io/) - these don't have wide adoption yet, but we recommend it.
+# You may well want to formalise this using the [codemeta.json](https://codemeta.github.io/) standard or the [citation file format](http://citation-file-format.github.io/).
 
 # %% [markdown]
 # ### Define packages and executables
 
 # %% [markdown]
 # We need to create `__init__` files for the source and the tests.
 # ```bash
-# touch greetings/greetings/test/__init__.py
-# touch greetings/greetings/__init__.py
+# touch greetings_repo/tests/__init__.py
+# touch greetings_repo/src/greetings/__init__.py
 # ```
```

Comment:
we don't need `__init__.py` anymore, and probably the problem with pytest is gone with this file structure as we won't be using relative imports, right?

Reply:
Oh yes, we'll need it to add things directly under package's namespace, but definitely not in the tests folder. I'll take it out!